src/expression.d

```diff
@@ -5332,6 +5332,9 @@ public:
                         return e.semantic(sc);
                     }
                 }
+                // ti is an instance which requires IFTI.
+                sds = ti;
+                type = Type.tvoid;
```

Comment:
This remains still problematic, b/c whenever we don't handle ScopeExp correctly, a wrong type (void) might slip through.
How about using `Type.terror` instead?